x/deployment/handler/handler_test.go

```diff
@@ -145,8 +148,17 @@ func TestUpdateDeploymentExisting(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, res)
 
+	t.Run("assert deployment version", func(t *testing.T) {
+		d, ok := suite.dkeeper.GetDeployment(suite.ctx, deployment.DeploymentID)
+		require.True(t, ok)
+		assert.Equal(t, d.Version, testutil.Sha256Nonce[:])
+	})
+
+	depSum := sha256.Sum256([]byte("ohai"))
```

Comment:
can you just use `testutil.Sha256Nonce`?  also, call it something else - `tetsutil.DefaultVersion`.

Reply:
The objective of the test was to run an update and confirm that the Version was updated with the new value. Thus the new Sum.

I set the second value to be the hash of the `DefaultDeploymentVersion` value, seem reasonable?